app-modules/interaction/src/Filament/Concerns/HasManyMorphedInteractionsTrait.php

```diff
@@ -100,17 +101,28 @@ public function table(Table $table): Table
             ->defaultSort('end_datetime', 'desc')
             ->columns([
                 TextColumn::make('subject')
+                    ->description(fn ($record) => $record->initiative->name . ' (' . $record->driver->name . ')')
                     ->icon(fn ($record) => $record->is_confidential ? 'heroicon-m-lock-closed' : null)
                     ->tooltip(fn ($record) => $record->is_confidential ? 'Confidential' : null),
+                TextColumn::make('type.name')
+                    ->label('Type')
+                    ->toggleable()
+                    ->sortable(),
+                TextColumn::make('status.name')
+                    ->label('Status')
+                    ->toggleable()
+                    ->sortable(),
                 TextColumn::make('start_datetime')
                     ->label('Start Time')
-                    ->dateTime(),
-                TextColumn::make('end_datetime')
-                    ->label('End Time')
-                    ->dateTime(),
+                    ->dateTime()
+                    ->sortable(),
                 TextColumn::make('created_at')
```

Comment:
It may be best to rename this to `duration` since the state is calculated and `created_at` is a legit column name.